src/common/badges-dashboard/BadgeMap.js

```diff
@@ -0,0 +1,5 @@
+export const BadgeMap = {
+  "Growling Panther": "winner",
```

Comment:
How do we plan to accomodate future badges in this structure?

For the upcoming event badges for example?

Reply:
removed